contributions/catalog/webapps/templates/contribute/contribution_details.html

```diff
@@ -410,10 +415,35 @@ <h4>Last Updated</h4>
 
 {% include 'topnav_javascript.html' %}
 
+<!-- Modal -->
+<div class="modal fade" id="confirmDelete" tabindex="-1" role="dialog" aria-labelledby="confirmDeleteTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="confirmDeleteTitle">Delete Contribution</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                Delete contribution : {{post.name}} ?
```

Comment:
```suggestion
                Are you sure you want to delete the Contribution {{post.name}}?
```
Suggesting a text change.